src/main/groovy/de/felixschulze/gradle/HockeyAppUploadTask.groovy

```diff
@@ -147,11 +153,13 @@ class HockeyAppUploadTask extends DefaultTask {
             }
         }
 
-        uploadAppplicationFileToHockeyApp(applicationFile, mappingFile, appId)
+        applicationFiles.each {
+            uploadApplicationFileToHockeyApp(it, mappingFile, appId)
```

Comment:
Ok than i will test the stuff next week and merge if all works fine :) Thanks for your input.